brave/brave-core
# Fix loading of Brave Ads resources in case of invalid resource

Resolves https://github.com/brave/brave-browser/issues/31106



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

Need to make sure that the visited page's text classification is working:

* Visit https://bitcoin.org
* Check that page was classified as crypto-crypto: `Classified text with the top segment as crypto-crypto` in Rewards log
